sklearn/cluster/_kmeans.py

```diff
@@ -813,11 +821,27 @@ def __init__(
         self.verbose = verbose
         self.random_state = random_state
 
-    def _check_params(self, X):
+    def _check_params(self, X, default_n_init):
```

Comment:
let's give it a default suche that it can be called from BisectingKMeans (the value doesn't matter since it will not use it)
```suggestion
    def _check_params(self, X, default_n_init=None):
```